client/src/styles/Register.scss

```diff
@@ -1,69 +1,74 @@
 .register {
-    max-width: calc(100% -  16px * 2);
     box-sizing: border-box;
-    margin: 10vh 16px 0;
-    padding: 1rem;
-    animation-name: scale-in;
-    animation-duration: 0.3s;
-    animation-fill-mode: forwards;
-    border-radius: 3px;
-    background-color: white;
-    opacity: 0;
+    max-height: calc(100vh - 89px);
+    overflow: hidden;
```

Comment:
In this case, we will have cropped register form for landscape phones. 
Could we try to set background-image to div#root?

Reply:
fixed